loader/0000-pr-policy.md

```diff
@@ -0,0 +1,67 @@
+# Quilt Loader PR Policy
+# Summary and Scope
+This RFC defines the criteria for merging Pull Requests (PR)s into Quilt Loader. It applies to any repositories managed by the Quilt Loader team, as well as RFCs under the scope of Loader.
+This RFC does not define policy for anything out of the scope of the Loader team, but we hope that other teams will structure their policies similarly.
+
+# Motivation
+This RFC intends to make the process of creating, reviewing, and merging PRs easier for both the Quilt Loader team and potential contributors.
+
+The goals for the PR process are:
+- **Democratic** -- Features are added to Loader through consensus of the team, and not through a benevolant dictator for life (BDFL) who has total control over the project.
+- **Quick** -- Tools should be in place to allow the team to shut down bikeshedding and say good enough on features if needed. However, this doesn't mean that pull requests should be forced through.
+- **Hands-off** -- Every Loader team member shouldn't have to approve every PR. The team as a whole has a responsibility to ensure PRs are reviewed in a timely manner, but no person should have a responsibility to view every change.
+
+It is **not a goal** of this PR to be fool-proof from bad actors within the Loader team abusing the system. Being on any team in Quilt requires trust, and bad actors should be removed instead of worked around.
+
+# Explanation
+## Initial Review Process
+A Pull Request is considered to have been approved when:
+- The Pull Request has recieved 2 approvals by members of the Loader team.
+- The Pull Request has no changes requested reviews by any members of the Loader team
+- Nobody in the Loader team has requested more time to look at the Pull Request
+
+Team members with outstanding changes requested reviews may dismiss their own reviews.
+
+Admins may dismiss outstanding reviews at their discretion, but should only do so if the team member cannot be contacted within a reasonable timeframe. Conflicts should *not* be resolved by an admin, but use 
+
+## Final Comment Period (FCP)
+- Begins immediately after the above critera is met
+- Lasts 2 days for small implementation changes, 5 days for major refactors, and 10 days for new API introductions or RFCs. Bugfixes have no FCP and are considered mergable immediately.
+- May be ended at any time by a team member if they have concerns.
+
+
+## Conflicts
+If the Loader team cannot agree on whether to merge a PR or deny it, there are steps they can take to avoid decsision paralysis. This should be a rare process--teams are encouraged to resolve these disagreements through talking the details out, but sometimes that isn't an option.
```

Comment:
```suggestion
If the Loader team cannot agree on whether to merge a PR or deny it, there are steps they can take to avoid decision paralysis. This should be a rare process--teams are encouraged to resolve these disagreements through talking the details out, but sometimes that isn't an option.
```